main.go

```diff
@@ -883,6 +884,13 @@ func (task *TaskRun) run() error {
 	if err != nil {
 		return WorkerShutdown(err)
 	}
+	// using NewWriterLevel(...) instead of NewWriter(...) to be explicit about
+	// compression level, so easy to change later
+	gzipLogWriter, err := gzip.NewWriterLevel(logFileHandle, gzip.DefaultCompression)
+	if err != nil {
+		return WorkerShutdown(err)
+	}
+	gzipLogWriter.Name = "live_backing.log"
```

Comment:
@gregarndt thought I'd double check - are you setting any other header fields in docker worker that I should also be setting? I assume I should set `Name` here like this in case someone extracts to a file (and I guess this is how the default filename gets chosen if they do). Let me know if this isn't right though!